HISTORY.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog ##
 
+### In Developemnt - (Dev time: 1 hour) ###
```

Comment:
You don't have to create a new block. Just include it as part of the previous block itself.

Basically we will be creating a new block for every release.